### DISCUSSION TITLE
Merry Christmas and Happy New Year 2023!

### DISCUSSION THREAD

**Discussion by jtmoon79** [MichaIng/DietPi#6022] [Q&A]
A belated Merry Christmas and Happy New Year to the DietPi Team!

> ```text
> - MOTD : ^
>         *#*        Wishing you a Merry Christmas
>        *###*       and a very Happy New Year 2023!
>       *#####*      We hope Christmas fills every corner
>      *#######*     of your home and your heart with joy.
>     *#########*
>    *Merry  Xmas*   All the best from the DietPi team =))
>   *#############*
> ─────────────────────────────────────────────────────```

**Comment by MichaIng** [comment]
Many thanks, best wishes from my end also via this channel 🙂 🙏.
